docs/workflows.md

```diff
@@ -637,3 +637,190 @@ Use the Workflow Id in any requests to Cancel or Terminate.
 - [How to set a Cron Schedule in Java](/java/reference-workflowoptions/#cronschedule)
 - [How to set a Cron Schedule in PHP](/php/distributed-cron)
 - [How to set a Cron Schedule in Typescript](/typescript/clients)
+
+## Schedules
```

Comment:
Do we really want to duplicate all this in concepts and also here?

Reply:
This is generated from the source files.
In the future, this will happen at build time and you won't see this duplication.